docs/source/installation.rst

```diff
@@ -213,9 +188,7 @@ In the ``anaconda prompt`` terminal:
 
 4. Install required packages:
 
-   1. :code:`conda install -c conda-forge pystan`
-   2. :code:`conda install -c conda-forge prophet`
-   3. :code:`conda install -c conda-forge scipy`
+   :code:`conda install -c conda-forge scipy`
```

Comment:
do we still need this?

Reply:
No idea. I was wondering too. Why is it here in the first place?

Reply:
I think its was there bcz of the 3 commands before. So as thexy are gone it can go also I think

Reply:
@fkiraly might know

Reply:
I have removed this step completely. Let me know if it should be kept and in which wording.

Reply:
yes should be fine as it was also not in the `.yaml` file.